packages/sdk-metrics/src/InstrumentDescriptor.ts

```diff
@@ -44,18 +49,11 @@ export interface InstrumentDescriptor {
   readonly type: InstrumentType;
   readonly valueType: ValueType;
   /**
-   * @experimental
+   * See {@link MetricAdvice}
    *
-   * This is intentionally not using the API's type as it's only available from @opentelemetry/api 1.7.0 and up.
-   * In SDK 2.0 we'll be able to bump the minimum API version and remove this workaround.
+   * @experimental
```

Comment:
Kept the `@experimental` tag since the upstream type is itself `@experimental`. If it's not worth it I can also just remove the doc comment altogether.

FWIW, this _entire_ interface is marked as `@deprecated` in `sdk-metrics/index.ts` in favor of `MetricDescriptor`, which does not have this field. But since @pichlermarc opened #5224 pretty recently, I assume the intention is to still keep this and carry it forward into 2.0.

Reply:
Keeping it experimental is good - we'll bump it to stable in a separate issue. 👍 

The `InstrumentDescriptor` is marked as deprecated as we will stop exporting it. Most properties will move to a `MetricDescriptor` and `InstrumentDescriptor` will extend it for internal use. Issue to follow.